### ISSUE TITLE
False positive for `Layout/IndentationConsistency` with `EnforcedStyle: indented_internal_methods`

### ISSUE THREAD

**Issue by kamipo** [rubocop-hq/rubocop#8919]
Looks like there is a regression for `Layout/IndentationConsistency` with `EnforcedStyle: indented_internal_methods` against Struct block in RuboCop 1.0.0.

## Steps to reproduce the problem

```yaml
# .rubocop.yml
AllCops:
  DisabledByDefault: true

Layout/IndentationConsistency:
  Enabled: true
  EnforcedStyle: indented_internal_methods
```

```ruby
# foo.rb
Foo = Struct.new(:foo) do
  private
    def private_foo
      foo
    end
end
```

```
% rubocop foo.rb
Inspecting 1 file
C

Offenses:

foo.rb:3:5: C: Layout/IndentationConsistency: Inconsistent indentation detected.
    def private_foo ...
    ^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

## Expected behavior

No offences.

## Actual behavior

```
% rubocop foo.rb
Inspecting 1 file
C

Offenses:

foo.rb:3:5: C: Layout/IndentationConsistency: Inconsistent indentation detected.
    def private_foo ...
    ^^^^^^^^^^^^^^^

1 file inspected, 1 offense detected, 1 offense auto-correctable
```

## RuboCop version

1.0.0 (using Parser 2.7.2.0, rubocop-ast 0.8.0, running on ruby 2.7.1 x86_64-darwin17)

**Comment by koic** [comment]
@kamipo This issue has been resolved by https://github.com/rubocop-hq/rubocop-ast/pull/141 and RuboCop AST 1.0.1 has been released. Can you update rubocop-ast to 1.0.1? Thank you.

**Comment by kamipo** [comment]
Thank you for the quick fix! I've confirmed that is fixed in https://github.com/rails/rails/pull/40436.
